tools/setup_helpers/dist_check.py

```diff
@@ -34,12 +43,16 @@ def should_build_ib():
     Helper function that detects the system's IB support and returns if we
     should build with IB support.
     """
+    ib_util_found = False
```

Comment:
Couldn't one conceivably want to build a binary with Infiniband support on a system that doesn't actually have a working ib-devinfo command, so that the binary could be shipped over? In that case, you wouldn't want this test anymore.

Reply:
I would want to be super safe about enabling IB, so in that case, he can do:

WITH_GLOO_IBVERBS=1 python setup install